.tag-details.pl

```diff
@@ -496,8 +500,9 @@ sub date {
 		say "-\t" . 'Layers:';
 		for my $layer (@{ $imageData->{layers} }) {
 			say "\t-\t" . '`' . $layer->{digest} . '`  ';
-			say "\t\t" . 'Last Modified: ' . date($layer->{lastModified}) . '  ';
-			say "\t\t" . 'Size: ' . size($layer->{size});
+			say "\t\t" . 'Last Modified: ' . date($layer->{lastModified}) . '  ' if defined $layer->{lastModified};
+			say "\t\t" . 'Size: ' . size($layer->{size}) . '  ' if defined $layer->{size};
+			say "\t\t" . 'MIME: ' . $layer->{mediaType};
```

Comment:
I figured we might as well include this now since it'll make it obvious when a layer is foreign. :smile: